packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.6
```

Comment:
Since there's no public API change, it should be 0.7.5+2

Reply:
done.

Reply:
I was a bit to fast, this doesn't look changed. It should have been `0.7.5+2`